packages/manager/apps/pci-ai-endpoints/public/translations/onboarding/Messages_fr_FR.json

```diff
@@ -0,0 +1,19 @@
+{
+  "title": "Enhance your applications with AI Endpoints",
+  "description": "Designed with simplicity in mind, our platform allows developers of all skill levels to enhance their applications with cutting-edge AI APIs — no AI expertise required.",
+  "description1": "These endpoints require",
+  "description2": "no AI expertise",
+  "description3": "or dedicated infrastructure, as the serverless platform provides",
+  "description4": "access to advanced AI models",
+  "description5": "including Large Language Models (LLMs), natural language processing, translation, speech recognition, image recognition, and more. Developers can select from a range of models, including",
+  "description6": "open-source options like Mistral AI, Llama and Stable Diffusion.",
```

Comment:
**nitpick:** This should be a single sentence. Breaking it will make it complicated for translations.

Reply:
Yep Fixed